.github/workflows/test-action-build-init.yml

```diff
@@ -102,3 +108,10 @@ jobs:
           # if fail-on-error == 'false', the grep should succeed (!fail) and never run the false command
           # short circuit if skip-init is 'true'
           ${{ matrix.skip-init }} || ! grep -- '--fail-on-error' conf.py || ${{ matrix.fail-on-error }} || exit 1
+
+          # check if provide-links-safely was used
+          # :orphan: and the labels mentioned in provide-links-safely should end up in rst/_targets.rst
+          # short circuit if skip-init is 'true'
+          ${{ matrix.skip-init }} || grep -- '^:orphan:$' rst/_targets.rst || exit 1
+          ${{ matrix.skip-init }} || grep -- '^.. _outside_reference_1:$' rst/_targets.rst || exit 1
+          ${{ matrix.skip-init }} || grep -- '^.. _outside_reference_2:$' rst/_targets.rst || exit 1
```

Comment:
```suggestion
          # short circuit if not provided
          ${{ matrix.provide-links-safely == '' }} || grep -- '^:orphan:$' rst/_targets.rst || exit 1
          ${{ matrix.provide-links-safely == '' }} || grep -- '^.. _outside_reference_1:$' rst/_targets.rst || exit 1
          ${{ matrix.provide-links-safely == '' }} || grep -- '^.. _outside_reference_2:$' rst/_targets.rst || exit 1
```

Reply:
Doesn't it need to short-circuit on both `matrix.skip-init` and `matrix.provide-links-safely == ''`?

Reply:
In theory, yes actually, I think we're always setting it to `''` when `skip-init` is true due to it being an include in the matrix maybe? I didn't look closely enough though, so I trust you on that. It'll be longer but probably better to do both.

Reply:
Implemented in 8d376978132dbfe3c030303e2565332a23cba043.